index.js

```diff
@@ -6,37 +6,93 @@ app.use(express.json());
 app.use(express.static('public'));
 app.set('view engine', 'ejs');
 
+const sessions = {};
+
 const opentok = require('./services/opentok-api');
 
+const giveMeCredentials = async (usertype, roomName) => {
```

Comment:
can we call this `generateCredentials` or something?

Reply:
sure thing 👍